src/test/java/org/example/homework13/blackjack/DeckTest.java

```diff
@@ -0,0 +1,41 @@
+package org.example.homework13.blackjack;
+
+import org.example.blackjack.Card;
+import org.example.blackjack.Deck;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeckTest {
+    private Deck deck;
+
+    @BeforeEach
+    void setUp() {
+        deck = new Deck();
+    }
+
+    @Test
+    public void testDrawCard() {
+        int initialSize = deck.size();
+        Card drawnCard = deck.drawCard();
+
+        assertNotNull(drawnCard);
+        assertEquals(initialSize - 1, deck.size());
+    }
+
+    @Test
+    public void testShuffle() {
```

Comment:
your test is not enough to check that shuffle is working. Check that first card is different after shuffling does not guarantee that the deck is shuffled the right way. By the way, there is a possibility that those cards could be equal
```suggestion
    public void testShuffle() {
            List<Card> originalCards = new ArrayList<>(deck.getCards());
        deck.shuffle();
        List<Card> shuffledCards = deck.getCards();
        assertEquals(originalCards.size(), shuffledOrder.size(),
                "The size of the deck should remain unchanged");
        assertNotEquals(originalCards, shuffledOrder,
                "The deck should be shuffled and not in original order");
        assertTrue(shuffledOrder.containsAll(originalCards) && originalCards.containsAll(shuffledOrder),
                "The shuffled deck should contain all original cards");
```